eng/test-mounts.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -xeuo pipefail
+IFS=$'\n\t'
+
+# We take the snap name as an input parameter
+snap="$1"
+dotnet_path="$2"
+content_snap_path="/snap/$snap/current"
+
+# Test .NET output
+"$dotnet_path"/dotnet --info
+
+echo "Installing .mount units..."
+cp "$content_snap_path"/mounts/* /usr/lib/systemd/system
+systemctl daemon-reload
+
+for unit_path in "$content_snap_path"/mounts/*.mount; do
+    # Remove the preceding path from the unit name
+    unit=$(echo "$unit_path" | rev | cut -d / -f 1 | rev)
```

Comment:
suggestion: You could also use the [basename(1)](https://manpages.ubuntu.com/manpages/oracular/en/man1/basename.1.html) command (e.g. `basename "$unit_path"`) to get the filename.

Reply:
note: with `basename --suffix=.mount "$unit_path"` would also remove the `.mount` suffix

Reply:
Ah, good to know. Will upload revision soon.